src/workspace/workspaceMonitor.ts

```diff
@@ -133,7 +136,7 @@ export class WorkspaceMonitor implements vscode.Disposable {
 		this.maybeNotifyOutdated(workspace);
 		this.maybeNotifyAutostop(workspace);
 		this.maybeNotifyDeletion(workspace);
```

Comment:
Was thinking about this some more, and we probably want to skip the deletion notification while in the setup phase too, otherwise I think we would get one notification here, and then another from the error thrown during setup.

Autostop and outdated I think are OK to show even during setup.